cmd/uhc/cluster/describe/describe.go

```diff
@@ -0,0 +1,135 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package describe
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/util"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client/clustersmgmt/v1"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug bool
+}
+
+var Cmd = &cobra.Command{
+	Use:   "describe CLUSTERID",
+	Short: "decribe a cluster",
+	Long:  "Get info about a cluster identified by its cluster ID",
+	Run:   run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+}
+func run(cmd *cobra.Command, argv []string) {
+
+	if len(argv) != 1 {
+		fmt.Fprintf(os.Stderr, "Expected exactly one cluster\n")
+		os.Exit(1)
+	}
+
+	clusterID := "id='" + argv[0] + "'"
```

Comment:
It is generally better to use `fmt.Sprintf` to format strings, as it requires less quoting:

```
clusterID := fmt.Sprintf("id='%s'", argv[0])
```

I'd also suggest to rename this variable, as it not really a cluster identifier, but rather a search criteria, or search query. Consider `searchQuery` or something similar.

You may also want to do some sanity check on the value, as it may be subject to SQL injection. I'd suggest, for example, to check that it only contains digits, letters and the `-` and `_` characters.

Not a big problem, however, as the server is SQL injection safe, and will never give you results that you can't see. I won't block you patch if you prefer to not do this.

Reply:
Is there anywhere else in the CLI that does sanitation that I could look at?

Reply:
No, there isn't, forget it, it is not that important.